docs/articles/index.html

```diff
@@ -0,0 +1,138 @@
+    <link rel="stylesheet" type="text/css" href="http://cdn3.devexpress.com/jslib/16.1.4/css/dx.common.css" />
+    <link rel="stylesheet" type="text/css" href="http://cdn3.devexpress.com/jslib/16.1.4/css/dx.ios7.default.css" />
+    <link href="css/gallery.css" type="text/css" rel="stylesheet">
+
+    <!--GitHub Star button-->
+    <script async defer src="https://buttons.github.io/buttons.js"></script>
+</head>
+<body>
+    <form id="main-form" action="#">
+    <div class="banner">
+        <div class="header">
+        TestCafe Open Source Edition
+        </div>
+        <div class="slogan">
+        Functional Web Testing Made Easy
+        </div>
+        <div class="white-button-container">
+        <a href="http://devexpress.github.io/testcafe/documentation/getting-started/" class="white-button">Get Started</a>
+        </div>
+        <a class="github-button" href="https://github.com/devexpress/testcafe/star" data-style="mega" data-count-href="/DevExpress/testcafe/stargazers" data-count-api="/repos/DevExpress/testcafe#stargazers_count" data-count-aria-label="# stargazers on GitHub" aria-label="Star DevExpress/testcafe on GitHub">Star</a>
+    </div>
+    <div class="main-content">
+        <div class="feature">
+            <span class="name">
+                Easy Install
+            </span>
+            <div class="description">
+                <p>All you need is to have <a href="https://nodejs.org">Node.js</a> with <a href="https://www.npmjs.com">npm</a> installed and call a single command.</p>
+                <div class="command">npm install --save-dev testcafe</div>
+                <p>No browsers plugins to install, no binary compilation post-install steps.</p>
+            </div>
+        </div>
+        <div class="feature">
+            <span class="name">
+                Complete Test Harness
+            </span>
+            <div class="description">
+                <p>TestCafe provides all testing features: starting browsers, running tests and gathering results. You configure test run and start execution either via the API or from a command shell simply by running a single command</p>
+                <div class="command">testcafe safari,ie tests/</div>
+                <p>TestCafe aggregates test results from different browsers and outputs them into one comprehensive report.</p>
+            </div>
+        </div>
+        <div class="feature">
+            <span class="name">
+                Write Test Code Using ES2016
+            </span>
+            <div class="description">
+                <p>TestCafe tests are written with ES2016 using all latest JavaScript features like 'async/await'. TestCafe will compile your test code on-flight and run it immediately. It also ships with built-in support for source maps to leverage debugging experience. Source maps are automatically enabled, so all you need to do is start a debugging session in an IDE that supports them.</p>
```

Comment:
> TestCafe tests are written with ES2016

I'm not sure about that but maybe use imperative mood here? Because tests are not yet written - they are to be written,

`Write TestCafe tests with ES2016...`

**BUT** I may be wrong and `test are written` is a common way to say it that suits here well - I just don't know it